pom.xml

```diff
@@ -515,6 +515,30 @@
                     <protocArtifact>com.google.protobuf:protoc:3.22.0:exe:${os.detected.classifier}</protocArtifact>
                 </configuration>
             </plugin>
+
+            <plugin>
+                <groupId>com.github.bohnman</groupId>
+                <artifactId>package-info-maven-plugin</artifactId>
+                <version>1.1.0</version>
+                <configuration>
+                    <!-- Couldn't get this to work for main and ext code at the same time. -->
```

Comment:
Sorry Joel for not looking at this before, but if you want 2 executions with different config, then you create two execution blocks and move the config inside the execution. This is the default, so moving only the  `sourceDirectory` should be enough. This is however good enough.